.coveragerc

```diff
@@ -443,6 +443,7 @@ omit =
     homeassistant/components/notify/xmpp.py
     homeassistant/components/nuimo_controller.py
     homeassistant/components/prometheus.py
+    homeassistant/components/remember_the_milk.py
```

Comment:
This does not match the file that you added

Reply:
Ah, yes thank you. I missed that when renaming the files...